stdeb.cfg

```diff
@@ -1,6 +1,6 @@
 [DEFAULT]
-Depends: python-six, python-systemd, python-requests, python-gevent
-Recommends: lldp, ips-reverseproxy-common, python-mdnsbridge, python-nmoscommon
-Depends3: python3-six, python3-systemd, python3-requests, python3-gevent
-Recommends3: lldp, ips-reverseproxy-common, python3-mdnsbridge, python3-nmoscommon
+Depends: ips-reverseproxy-common, python-six, python-systemd, python-requests, python-gevent, python-mdnsbridge, python-nmoscommon
+Recommends: lldp
+Depends3: ips-reverseproxy-common, python3-six, python3-systemd, python3-requests, python3-gevent, python3-mdnsbridge, python3-nmoscommon
```

Comment:
This was clearly already an issue, but the reverse proxy package name changes for python3 to 'python3-nmosreverseproxy'. Would you mind correcting that as part of these PRs?

Reply:
sure

Reply:
Done